vue/src/vueTextMask.js

```diff
@@ -0,0 +1,49 @@
+import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
+
+/*
+ * Vue Directive - Text Masking with Vanilla Text Mask
+ * https://github.com/text-mask/text-mask/tree/master/vanilla
+ *
+ * This directive binds input masking via Text Mask. Once the element is
+ * destroyed it will be unbinded. Pass an options object, same as you would
+ * normally, as a parameter on the element being bound.
+ */
+
+/**
+ * Initialize variable to store directive's `this` so that we can use its
+ * context in inputHandler called via event listener
+ */
+let instance
+
+// Register the input handler to watch for updates
+const inputHandler = ({target: {value}}) => {
+  return instance.textMaskInputElement.update(value)
+}
```

Comment:
I think you should move this to inside the directive and attach it to the directive, too.